From 8114d9f57a9594901898bcfff47f02fb0032554e Mon Sep 17 00:00:00 2001
From: James Amattey <me@jamattey.com>
Date: Mon, 14 Feb 2022 02:31:47 +0000
Subject: [PATCH] Added

---
 client_test.py | 56 +++++++++++++++++++++++++-------------------------
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/client_test.py b/client_test.py
index 6c61c88..740a1c4 100644
--- a/client_test.py
+++ b/client_test.py
@@ -17,8 +17,8 @@ class ClientTest(unittest.TestCase):
     ]
     """ ------------ Add the assertion below ------------ """
 
-+ for quote in quotes;
-+   self.assertEqual(getDataPoint(quote), (quote['stock'], quote['top_bid']['price'] + quote['top_ask']['price'], quote['top_bid']['price'] + quote['top_ask']['price'])/2))
+   for quote in quotes;
+     self.assertEqual(getDataPoint(quote), (quote['stock'], quote['top_bid']['price'] + quote['top_ask']['price'], quote['top_bid']['price'] + quote['top_ask']['price'])/2))
 
 
 
@@ -32,37 +32,37 @@ class ClientTest(unittest.TestCase):
 
     """ ------------ Add the assertion below ------------ """
 
-  + for quote in quotes:
+     for quote in quotes:
     self.assertEqual(getDataPoint(quote), (quote['stock'], quote['top_bid']['price'] + quote['top_ask']['price'], quote['top_bid']['price'] + quote['top_ask']['price'])/2))
 
 
     """ ------------ Add unit tests ------------ """
 
-    + def test_getRatio_priceBZero(self):
-    + price_a = 119.2
-    + price_b = 0
-    + self.assertIsNone (getRatio (price_a, price_b))
-
-    + def test_getRatio_priceAZero(self):
-    + price_a = 0
-    + price_b = 121.68
-    + self.assertEqual (getRatio (price_a, price_b))
-
-
-    + def test_getRatio_greaterThan1(self):
-    + price_a = 346.48
-    + price_b = 166.39
-    + self.assertGreater (getRatio (price_a, price_b), 1)
-    +
-    + def test_getRatio_LessThan1(self):
-    + price_a = 166.39
-    + price_b = 356.48
-    + self.assertLess (getRatio (price_a, price_b), 1)
-    + 
-    + def test_getRatio_exactlyOne(self):
-    + price_a = 356.48
-    + price_b = 356.48
-    + self.assertEqual (getRatio (price_a, price_b))
+     def test_getRatio_priceBZero(self):
+     price_a = 119.2
+     price_b = 0
+     self.assertIsNone (getRatio (price_a, price_b))
+
+     def test_getRatio_priceAZero(self):
+     price_a = 0
+     price_b = 121.68
+    +self.assertEqual (getRatio (price_a, price_b))
+
+
+     def test_getRatio_greaterThan1(self):
+     price_a = 346.48
+     price_b = 166.39
+     self.assertGreater (getRatio (price_a, price_b), 1)
+    
+     def test_getRatio_LessThan1(self):
+     price_a = 166.39
+     price_b = 356.48
+     self.assertLess (getRatio (price_a, price_b), 1)
+     
+     def test_getRatio_exactlyOne(self):
+     price_a = 356.48
+     price_b = 356.48
+     self.assertEqual (getRatio (price_a, price_b))
 
 
 
-- 
2.34.1

